src/core/Akka.Remote/Serialization/ExceptionSupport.cs

```diff
@@ -84,14 +85,22 @@ public Proto.Msg.ExceptionData ExceptionToProtoNet(Exception exception)
             message.InnerException = ExceptionToProto(exception.InnerException);
 
             var serializable = exception as ISerializable;
-            var serializationInfo = new SerializationInfo(exceptionType, DefaultFormatterConverter);
+            var serializationInfo = new SerializationInfo(exceptionType, _defaultFormatterConverter);
             serializable.GetObjectData(serializationInfo, new StreamingContext());
 
             foreach (var info in serializationInfo)
             {
-                if (DefaultProperties.Contains(info.Name)) continue;
-                var preparedValue = _wrappedPayloadSupport.PayloadToProto(info.Value);
-                message.CustomFields.Add(info.Name, preparedValue);
+                if (_defaultProperties.Contains(info.Name)) continue;
+                if (info.Value is Exception exceptionValue)
+                {
+                    var exceptionPayload = ExceptionToProto(exceptionValue);
```

Comment:
Old code would fail if it encounters an Exception property inside an Exception, have to make sure that the serializer recurse through these